images/git-custom-k8s-auth/cloudbuild.yaml

```diff
@@ -20,4 +20,5 @@ steps:
     - gcr.io/$PROJECT_ID/git-custom-k8s-auth:latest
 substitutions:
   _GIT_TAG: '12345'
+  # Prefer 0.5.x series (not 0.6.x) for now.
```

Comment:
Do you want to comment why and/or when we would want to advance?

Reply:
I'm out of my depth for the 0.5 vs 0.6 difference. I think @ameukam might know, because of https://github.com/kubernetes/test-infra/pull/31725

Reply:
From slack discussion w/ @ameukam: `Something was odd. No release for 0.6.0`

Reply:
Currrently decided to stick with the `0.5.x` versions. There is no significant benefits to go to `0.6.x`.